lib/settings.js

```diff
@@ -10,6 +10,7 @@ const sourcePath = './assets/Zed/';
 const publicPath = '/assets/';
 const sourceDir = path.resolve(sourcePath);
 const publicDir = path.resolve(path.join('public/Backoffice', publicPath));
+const mpPublicDir = path.resolve(path.join('public/MerchantPortal', `${publicPath}/js/`));
```

Comment:
I'd suggest to not add `js` here as long as a the end there can be something else.